packages/backend-ai/src/get_message.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  BedrockRuntimeClient,
+  ConverseCommand,
+  ConverseCommandInput,
+  ConverseCommandOutput,
+  Message,
+} from '@aws-sdk/client-bedrock-runtime';
+import {
+  ContentBlock,
+  GetMessageInput,
+  GetMessageOutput,
+  Tool,
+} from './types.js';
```

Comment:
Can we remove the `.js` ?

Reply:
I think when when '--moduleResolution' is 'node16' or 'nodenext', TypeScript enforces stricter rules for module resolution that align with how Node.js handles ES modules. Therefore `.js` is needed in this context